frontend/app/src/components/ui/badge.tsx

```diff
@@ -14,6 +14,8 @@ const badgeVariants = cva(
         destructive:
           'border-transparent bg-destructive text-destructive-foreground shadow hover:bg-destructive/80',
         outline: 'text-foreground',
+        outlineDestructive:
+          'border border-destructive rounded-sm px-1 font-normal text-red-800 dark:text-red-300 bg-transparent',
```

Comment:
![image](https://github.com/user-attachments/assets/33d093a2-ba9f-4311-85f5-8f9d16042aef)
![image](https://github.com/user-attachments/assets/efcd52cc-0fcf-48d1-8fb3-0831fc47962f)

Reply:
ui nit: do you think we can make the status buttons grow so they're all the same width?